python/test/unit/language/test_core.py

```diff
@@ -5242,3 +5242,67 @@ def test_tl_range(device):
             ptx = pgm.asm['ptx']
             # check that the loop got pipelined with the right number of stages.
             assert 'cp.async.wait_group 0x6' in ptx
+
+
+def test_descriptor_load_ttgir():
```

Comment:
I don't think we should move this in test_core, this is meant to test experimental features I would prefer this to stay in a different file. Leaving them in hopper/ or language/ is fine but it should still be in a separate file with the existing descriptive name

Reply:
OK, I will put it in a separate file.